src/components/Book.js

```diff
@@ -12,9 +12,11 @@ function Book({
     if (authors.length === 2) {
       if (authors[0].includes(authors[1])) {
```

Comment:
I was a little bit puzzled about this line of code, but after checking on your data.json file I got the point of what you're checking here for: You provide an option to set an authors name that might be very long to a short version and then check whether both names are present in the data and if so, only return the long one.

While I find that to be a very neat feature I can see that to cause potentially larger , albeit theoretical, problems:

First, imagine having a Book written by two different authors who share names (or parts of it) however: John Doe Smith and John Doe.
Your current condition would return only "John Doe Smith", as the second authors name will be read as an alias of the first author.

Secondly, consider pseudonyms: One Book might be shown as written by two authors, J.K. Rowling and Robert Galbraith, for example, when in fact, they are the same person.

One solution that comes to my mind to solve this issue might be to put each author in your data.json in an individual array themselves, where you then again can put short name versions and aliases/pseudonyms, like so:
```
[...]
"authors": [
    ["Ildefonso Falcones de Sierra", "Ildefonso Falcones"],
    ["J.K. Rowling", "RobertGalbraith"]
[...]
```

Your function would then look like this:
```
function authorsLength(authors) {
    if (authors.length === 1) {
        return `${authors[0][0]}`        // only the first entry to the author
        return `${authors[0]}`             // authors name and all his aliases/pseudonyms
    }
}   
```
Based on the returned array you could then display aliases/pseudonyms as well.

Reply:
Thank you for your reply! I haven't heard about the `ellipsis `functionality in CSS yet! Will check. :-)